src/Parsing/String.purs

```diff
@@ -195,6 +195,9 @@ match p = do
 
 -- | Compile a regular expression `String` into a regular expression parser.
 -- |
+-- | Note that per JS RegExp semantics matching a single line in a multiline
```

Comment:
This is good, thanks, but would you please put it down below in the **Flags** section than right up at the top of the docs.

Reply:
Done!